privacyidea/lib/tokens/hotptoken.py

```diff
@@ -690,27 +690,23 @@ def get_default_settings(cls, params, logged_in_user=None,
             return ret
         (role, username, userrealm, adminuser, adminrealm) = determine_logged_in_userparams(logged_in_user,
                                                                                             params)
-        hashlib_pol = policy_object.get_action_values(
-            action="hotp_hashlib",
-            scope=role,
-            user=username,
-            realm=userrealm,
-            adminrealm=adminrealm,
-            adminuser=adminuser,
-            client=client_ip,
-            unique=True)
+        hashlib_pol = Match.generic(g, scope=role,
+                                    action="hotp_hashlib",
```

Comment:
check if we can use constants here

Reply:
I checked it. There is no ACTION constanct "hashlib" and no constant "hotp_hashlib".
https://github.com/privacyidea/privacyidea/blob/master/privacyidea/lib/tokens/hotptoken.py#L170

So I will not do this here.

Reply:
Ah, this is still in my git stash :-)